<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QLabel" name="paramChangeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Param</string>
   </property>
  </widget>
  <widget class="QLabel" name="activeCompLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Active component</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reloadQss_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reload QSS</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
